common/src/main/java/com/wynntils/core/encoding/EncodedByteBuffer.java

```diff
@@ -4,25 +4,114 @@
  */
 package com.wynntils.core.encoding;
 
-// byte[] <-> UTF-16, Base64, etc.
+import com.wynntils.utils.type.UnsignedByte;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+
+/**
+ * A buffer of bytes that can be encoded and decoded to various formats.
+ */
 public final class EncodedByteBuffer {
```

Comment:
I must have missed when we got this class. It feels more "utils" than "core" to me.

Reply:
This PR targets a separate branch for gear encoding, so it doesn't exist on main.

Reply:
I've done some basic modeling in the PR this class was added, but I can agree with you. The main reason I put it in core is because I plan to use this class throughout Artemis for encoding different kind of data (like POIs maybe).

Reply:
I think it's best leaving it here until the original branch is ready and check back on this. If I add nothing else here, I am going to move it.

Reply:
I ended up moving the class here.